snippets/algo_snippets.json

```diff
@@ -44,6 +44,386 @@
             "\t\telse:\n",
             "\t\t\treturn ans * ans * x\n"
         ],
-        "description":"peforms a^b operation with O(log(n)) Complexity"
+        "description": "peforms a^b operation with O(log(n)) Complexity"
```

Comment:
good works, tell me when you're ready to merge and tell me when you think it's enough to update the extension

Reply:
I want to add more graph algorithms like `Dijkstra's`, 'BFS`, `DFS` but they are a little bit complicated and every one might have there own implementation variation , so for now we can restrict our self for these algorithms

Reply:
Sure. So merge it this Sunday? Or u want to continue check first?

Reply:
if you feel everything is alright merge this pull request into your repo `now` so that we can check `GitHub Action`, on Sunday publish a `new version` of this extension to vscode marketplace, does this work?

Reply:
> if you feel everything is alright merge this pull request into your repo `now` so that we can check `GitHub Action`, on Sunday publish a `new version` of this extension to vscode marketplace, does this work?

I haven't check the code yet, i'm a bit busy this week. I'll merge it now but keep an eye on any typo u want to change before Monday. I will be able to update the extension hopefully next monday.